### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by adamrturman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,16 +20,16 @@ npm install @kava-labs/javascript-sdk
 
 ### Mainnet
 
-- Chain ID: kava-3
-- REST API endpoint: https://kava3.data.kava.io
+- Chain ID: kava-7
+- REST API endpoint:  https://api.kava.io
 - BEP3 deputy's Kava address: kava1r4v2zdhdalfj2ydazallqvrus9fkphmglhn6u6
```
Comment (denalimarsh) [inline README.md:25]: Add Kava deputy addresses for all assets as seen in https://github.com/Kava-Labs/javascript-sdk/pull/55

File: README.md [hunk 2]
```diff
@@ -20,16 +20,16 @@ npm install @kava-labs/javascript-sdk
...
 - BEP3 deputy's Binance Chain address: bnb1jh7uv2rm6339yue8k4mj9406k3509kr4wt5nxn
```
Comment (denalimarsh) [inline README.md:26]: Add Binance Chain deputy addresses for all assets as seen in https://github.com/Kava-Labs/javascript-sdk/pull/55

File: README.md [hunk 3]
```diff
@@ -20,16 +20,16 @@ npm install @kava-labs/javascript-sdk
...
 - Binance Chain mainnet REST API endpoint: https://dex.binance.org/
 
 ### Testnet
 
-- Chain ID: kava-testnet-8000
-- REST API endpoint: https://kava-testnet-8000.kava.io
+- Chain ID: kava-testnet-12000
+- REST API endpoint: https://api.data-testnet-12000.kava.io
 - BEP3 deputy's Kava address: kava1tfvn5t8qwngqd2q427za2mel48pcus3z9u73fl
```
Comment (denalimarsh) [inline README.md:33]: Update testnet deputy address (or remove)

File: README.md [hunk 4]
```diff
@@ -20,16 +20,16 @@ npm install @kava-labs/javascript-sdk
...
 - BEP3 deputy's Binance Chain testnet address: tbnb1mdvtph9y0agm4nx7dcl86t7nuvt5mtcul8zld6
```
Comment (denalimarsh) [inline README.md:34]: Update testnet deputy address (or remove)

**Review by denalimarsh — CHANGES_REQUESTED** [review]
Please also update the example scripts with the new endpoints as done in https://github.com/Kava-Labs/javascript-sdk/pull/55.

File: README.md [hunk 5]
```diff
@@ -20,20 +20,48 @@ npm install @kava-labs/javascript-sdk
 
 ### Mainnet
 
-- Chain ID: kava-3
-- REST API endpoint: https://kava3.data.kava.io
+- Chain ID: kava-7
+- REST API endpoint:  https://api.kava.io
 - BEP3 deputy's Kava address: kava1r4v2zdhdalfj2ydazallqvrus9fkphmglhn6u6
```
Comment (denalimarsh) [inline README.md:25]: Can remove this line now that deputy addresses are added below

File: README.md [hunk 6]
```diff
@@ -20,20 +20,48 @@ npm install @kava-labs/javascript-sdk
 
 ### Mainnet
 
-- Chain ID: kava-3
-- REST API endpoint: https://kava3.data.kava.io
+- Chain ID: kava-7
+- REST API endpoint:  https://api.kava.io
```
Comment (denalimarsh) [inline README.md:24]:
```suggestion
- REST API endpoint: https://api.kava.io
```
Extra space here

File: README.md [hunk 7]
```diff
@@ -20,20 +20,48 @@ npm install @kava-labs/javascript-sdk
...
 - BEP3 deputy's Kava address: kava1r4v2zdhdalfj2ydazallqvrus9fkphmglhn6u6
 - BEP3 deputy's Binance Chain address: bnb1jh7uv2rm6339yue8k4mj9406k3509kr4wt5nxn
```
Comment (denalimarsh) [inline README.md:26]: Likewise, can remove this line now that deputy addresses are below

**Review by denalimarsh — CHANGES_REQUESTED** [review]

**Review by denalimarsh — APPROVED** [review]
Changes LGTM 👍

File: README.md (hunk 7 above)
Comment (adamrturman) [reply README.md]: fixed in [7a7494d826c34ffc25fce18df8c2e9de88059dbe](https://github.com/Kava-Labs/javascript-sdk/commit/7a7494d826c34ffc25fce18df8c2e9de88059dbe)

**Pull request merged**